logging/bin/deploy_fluentbit_azmonitor.sh

```diff
@@ -140,6 +168,7 @@ FLUENTBIT_HELM_CHART_VERSION=${FLUENTBIT_HELM_CHART_VERSION:-"0.24.0"}
 helm $helmDebug upgrade --install v4m-fbaz  --namespace $LOG_NS  \
   --version $FLUENTBIT_HELM_CHART_VERSION \
   --values logging/fb/fluent-bit_helm_values_azmonitor.yaml \
+  --values $airgapValuesFile \
```

Comment:
Don't we need/want to reference $AIRGAP_HELM_REPO below (on line 173)?   Might be good to split line 173 below so the Helm chart name sits on its own line.

Reply:
We do need to add the $AIRGAP_HELM_REPO ... I might've missed this one because it was kind of flushed in like that.  I moved it to its own line and added the $AIRGAP_HELM_REPO parameter to it.  It will show up in the next push.